packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -686,6 +687,53 @@ describe('TransactionController', () => {
       );
     });
 
+    it.each([
+      [
+        'adds single unapproved transactions with same actionId',
```

Comment:
```suggestion
        'does not add duplicate transaction if actionId already used',
```

Reply:
I moved the tests to their own describe, so once I tried to use the commit suggestion it does not work, but I've included this improvement manually.